specs/master/cluster-init/scripts/030_install_scalelib.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+HF_TOP=$( jetpack config symphony.hostfactory.top )
+if [ -z "${HF_TOP}" ]; then
+   # In Symphony 7.2 and earlier: HF_TOP=$EGO_TOP/eservice/hostfactory
+    HF_TOP=$EGO_TOP/hostfactory
+fi
+
+HF_VERSION=$( jetpack config symphony.hostfactory.version )
+if [ -z "${HF_VERSION}" ]; then
+    HF_VERSION="1.1"
+fi
+
+PKG_NAME=$( jetpack config symphony.pkg_plugin )
+if [ -f $PKG_NAME ]; then
+    rm -f $PKG_NAME
+    rm -rf packages
+    rm -rf hostfactory
+fi
+
+jetpack download --project symphony symphony/$PKG_NAME | exit 1;
```

Comment:
Didn't we already do all of this in the new chef recipe?

Reply:
yes we did actually I was not extracting to a directory so was not sure but now will extract zip file to /tmp so can just refer that